src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -157,3 +158,22 @@ const natureOfControlTypeMap = new Map<string, NatureOfControlType>([
   [natureOfControl.SIGNIFICANT_INFLUENCE_OR_CONTROL_AS_FIRM, NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL]
 ]);
 
+export const mapPrivateAddress = (boData: BeneficialOwnersPrivateData, ch_reference: string) => {
```

Comment:
could we name this mapBoPrivateAddress please? Or mapBeneficialOwnerPrivateAddress